packages/data-layer/package.json

```diff
@@ -1,7 +1,6 @@
 {
   "name": "data-layer",
   "version": "1.0.0",
-  "type": "module",
```

Comment:
how come this is necessary?

Reply:
Because the other apps in the repo aside from `verify-env` are not ESM and cross-importing leads to import errors.